app/services/core_data_connector/import/web_identifiers.rb

```diff
@@ -0,0 +1,130 @@
+          FROM   #{table_name} z_web_identifiers
+          WHERE  z_web_identifiers.web_identifier_id IS NULL
+          RETURNING id AS web_identifier_id, z_web_identifier_id
+          )
+          UPDATE #{table_name} z_web_identifiers
+              SET web_identifier_id = insert_web_identifiers.web_identifier_id
+             FROM insert_web_identifiers
+            WHERE insert_web_identifiers.z_web_identifier_id = z_web_identifiers.id
+        SQL
+      end
+
+      def transform
+        super
+
+        execute <<-SQL.squish
+          WITH related_types AS (
+          SELECT id, uuid, 'CoreDataConnector::Instance' AS type
+            FROM core_data_connector_instances instances
+           WHERE instances.z_instance_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Item' AS type
+            FROM core_data_connector_items items
+           WHERE items.z_item_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Organization' AS type
+            FROM core_data_connector_organizations organizations
+           WHERE organizations.z_organization_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Person' AS type
+            FROM core_data_connector_people people
+           WHERE people.z_person_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Place' AS type
+            FROM core_data_connector_places places
+           WHERE places.z_place_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Taxonomy' AS type
+            FROM core_data_connector_taxonomies taxonomies
+           WHERE taxonomies.z_taxonomy_id IS NOT NULL
+          UNION
+          SELECT id, uuid, 'CoreDataConnector::Work' AS type
+            FROM core_data_connector_works works
+           WHERE works.z_work_id IS NOT NULL
+          )
+          UPDATE #{table_name} z_web_identifiers
+             SET web_identifier_id = web_identifiers.id
+            FROM related_types, core_data_connector_web_identifiers web_identifiers
+           WHERE related_types.uuid = z_web_identifiers.identifiable_uuid
+             AND related_types.type = z_web_identifiers.identifiable_type
+             AND web_identifiers.identifier = z_web_identifiers.identifier
+             AND web_identifiers.web_authority_id = z_web_identifiers.web_authority_id
+        SQL
+      end
+
+      protected
+
+      def column_names
+        [{
+          name: 'web_authority_id',
+          type: 'INTEGER',
+          copy: true
+        }, {
+          name: 'identifiable_uuid',
+          type: 'UUID',
+          copy: true
+        }, {
+          name: 'identifiable_type',
+          type: 'VARCHAR',
+          copy: true
+        }, {
+          name: 'identifier',
+          type: 'VARCHAR',
+          copy: true
+        }, {
+          name: 'web_identifier_id',
+          type: 'INTEGER'
+        }, {
+          name: 'identifiable_id',
```

Comment:
Is this column being populated anywhere? I see we're inserting the column in the `load` method, but I'm not seeing the value being set anywhere.